src/pages/rest/tutorials/image/update.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Update an image 
+contributor_name: Goivvy LLC
+contributor_link: https://www.goivvy.com/
+description: We will learn how to update an existing image 
+--- 
+
+# Update an image
+
+We will update an image with **id=694**. We got image IDs in a previous [step](https://developer.adobe.com/commerce/webapi/rest/tutorials/image/list/).
+
+Here is a new image:
+
+![New Image](../../../_images/update-new-image.png)
+
+Let us get a **base64-encoded** string of the image:
+
+```bash
+base64 -i ~/update-new-image.png
+iVBORw0KGgoAAAANSUhEUgAAARwAAAFICAYAAACC60EBAAAKrGlDQ1BJQ0MgUHJvZmlsZQAASImVlwdUU8kax+fem16AkBABKaE3QToBpITQQpcONkISIJQQA0HFriwu4FpQEcGKLooouBZA1oootkWxANYNsqgo62LBhsq7wCG4+85777zvnDnzu//7zTffzJm557sAUNX4UmkmrAZAliRXFhngw4pPSGThnwIyIAIacAYIX5Aj5UREhADUJvq/2/tOAI32t61HY/37+/9qNKEoRwAAFIFysjBHkIXycbS9EEhluQAgu1HdaEGudJRbUWbI0ARR7h7l1HEeGOXkMcaAMZ/oSC7KDAAIFD5flgoAhYXqrDxBKhqH4o2yrUQolqAsRdkzKytbiPIRlM1RH1SjjMZnJ38XJ/VvMZOVMfn8VCWPr2XMCL7iHGkmf9H/uR3/27Iy5RNzmKKNkiYLjER7dXTPujOyg5UsSQ4Ln2CxcMx/jNPkgTETLMjhJk6wkO8brBybGRYywSlif54yTi4veoJFOX5REyzLjlTOlSLjciaYL5ucV54Ro9TTRDxl/Py06LgJzhPHhk1wTkZU8KQPV6nL5JHK/EWSAJ/Jef2Va8/K+W69Yp5ybG5adKBy7fzJ/EUSzmTMnHhlbkKRr9+kT4zSX5rro5xLmhmh9BdlBij1nLwo5dhc9EBOjo1Q7mE6PyhiggEXZINMtMkAC4SgT74A5IoW5o4uhJstXSQTp6blsjjoDROxeBKBzTSWva29PQCj93X8OLxljt1DiHl1UstqQo8xgt6RokkteRAAVAa01knN1BoA1a8AnFYI5LK8cW3sLmEBCagCBtACesAImANrYI9....[more characters]
+```
+
+An API call:
+
+**Endpoint:**
+
+```html
+PUT http://domain.com/rest/default/V1/products/MJ03/media
```

Comment:
This call needs to have the `id` added to the end of the path. `.../MJ03/media/{id}`. You should also mention that the `id` on the sample data images might not match the `id` listed in this tutorial, so they need to refer back to the list call.